frontend/src/routes/BareMetalAssets/BareMetalAsset.test.tsx

```diff
@@ -14,24 +19,13 @@ import {
 import { Secret, SecretApiVersion, SecretKind } from '../../resources/secret'
 import { ResourceAttributes } from '../../resources/self-subject-access-review'
 import BareMetalAssetsPage from './BareMetalAssetsPage'
-import { nockDelete, nockCreate, nockRBAC } from '../../lib/nock-util'
-import { Scope } from 'nock/types'
-import { RecoilRoot } from 'recoil'
-import { bareMetalAssetsState } from '../../atoms'
-import { clickByLabel, clickByRole, clickByText, waitForNock, waitForNocks, waitForText } from '../../lib/test-util'
 
 const bareMetalAsset: BareMetalAsset = {
     apiVersion: 'inventory.open-cluster-management.io/v1alpha1',
     kind: 'BareMetalAsset',
-    metadata: {
-        name: 'test-bare-metal-asset-001',
-        namespace: 'test-bare-metal-asset-namespace',
-    },
+    metadata: { name: 'test-bare-metal-asset-001', namespace: 'test-bare-metal-asset-namespace' },
```

Comment:
Why the formatting change to the metadata?

Reply:
i just find it easier to read, but i understand where you are coming from